students/ChelseaSmith/Lesson3/slicingfunctions.py

```diff
@@ -0,0 +1,23 @@
+def flip_first_last(seq):
+    orig = tuple(seq)
+    new = list(seq)
+    new[0] = orig[-1]
+    new[-1] = orig[0]
+    return new
```

Comment:
you should only use slicing for this assignment; right now it is using indexing.

here is how you would do it using slicing:
```
def flip_first_last(seq):
    return seq[-1:] + seq[1:-1] + seq[:1]
```